component/component.css

```diff
@@ -31,6 +31,7 @@
     /* *************************************** */
     /* Border radius  */
     --border-radius: 0.8rem;
+    --border-radius-textbox: 0.6rem;
```

Comment:
Try to refrain from using `rem` for small decimal values
Ideally use `px` units for anything less than 10-12px

Reply:
Yes, from next time will take of it. Thank you